RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -989,6 +994,11 @@ string banisherCombatString(monster enemy, location loc, boolean inCombat)
 		return "";
 	}
 
+	if(in_pokefam())
+	{
+		return false;
```

Comment:
```suggestion
		return "";
```
this function returns a string not a boolean. need to return "" to indicate not using it

Reply:
Looks good now. I think this is the only thing that still needs fixing and then it is ready in my opinion

Reply:
Thanks again, taltamir.  It took me two tries (lol) but I changed this